src/main/java/emu/grasscutter/commands/PlayerCommands.java

```diff
@@ -176,6 +177,110 @@ public void execute(GenshinPlayer player, List<String> args) {
             }
         }
     }
+
+    @Command(label = "givechar", aliases = { "givec" }, usage = "Usage: givechar <playerId> <avatarId> [level]")
+    public static class GiveCharCommand implements CommandHandler {
+        @Override public void execute(GenshinPlayer player, List<String> args) {
+            int target, avatarID, level = 1;
+
+            if(args.size() < 2) {
+                CommandHandler.sendMessage(null, "Usage: give <player> <avatarId> [level]");
```

Comment:
`give` instead of `givechar`?

Reply:
Maybe make the player variable optional when the player is submitting the command

Reply:
> `give` instead of `givechar`?

Good catch, that's fixed!

Reply:
> Maybe make the player variable optional when the player is submitting the command

makes sense